proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,258 @@
+3. Amendments to the association’s purpose may only be made in accordance with
+   §3 (1).
+
+## 3. Non-profit Association
+
+1. The Association’s activities are exclusively non-profit according to the
+   the criteria for tax-privileged entities in accordance with German Tax
+   Regulations. The Association and its activities are politically, ethnically
+   and denominationally neutral. They are for the common good and not for profit.
+2. Financial means may only be used for purposes stipulated in the Articles of
+   Association. Members are reimbursed for expenses but do not receive direct
+   benefits from the Association’s funds.
+3. It is not permitted to favor someone through association expenditures that
+   do not serve the association's purpose or are disproportionately high.
+
+## 4. Types of Membership and Members of the Association
+
+1. Membership is open to natural persons and legal bodies supporting the
+   association's aims. Membership types include:
+   1. **Core members**: Natural persons actively supporting the Association’s
+      purpose and aims. Primarily, they are expected to cooperate and contribute,
+      participate in the general assembly and exercise their voting rights.
+   2. **Extraordinary members**: Natural persons and legal bodies supporting the
+      Association’s aims but without voting rights and obligations to cooperate.
+      This gives members the opportunity to pause their cooperation and contribution
+      to Mixxx without necessarily losing their membership.
+   3. **Supporting Members**: Extraordinary members primarily supporting through
+      financial or material contributions. They have the right to participate in
+      the General Assemblies without voting rights.
+2. Core membership is granted upon suggestion or application by the General
+   Assembly or through an internet vote of the core members. Criteria include
+   long-term commitment and contributions.
+3. A core members can change status to that of an extraordinary members by
+   submitting their request to the Board. Should an active member not fulfil the
+   above mentioned duties over a period of two consecutive general meetings, the
+   membership will automatically change into an extraordinary membership.
+4. Membership terminates due to withdrawal, expulsion, death, or loss of legal
+   status. Withdrawal from the association shall take effect at the end of the
+   following calendar month in which the declaration of withdrawal was
+   submitted.
+5. The expulsion of a member may be decided by the General Assembly or by
+   Internet vote in the event of a critical situation where an amicable solution
+   cannot be reached. The following reasons may cause an expulsion:
+   a) Serious violations of regulations in the Articles of Association,
+      the purpose and aim of the Association or the Code of Conduct.
+   b) non-paiment of membership fees over a period of 12 months despite repeated
+      reminders.
```

Comment:
these subheads are not rendered correctly in markdown.

I don't think we need to mention reminders in part b. We should definitely get a lawyer to look over this whole thing just to tidy up the language.

```suggestion
   a) Serious violations of regulations in the Articles of Association,
      the purpose and aim of the Association or the Code of Conduct.
   b) non-payment of membership fees over a period of 12 months.
```